Added a card in Text Formatting - Text Tune

nelsonuprety1 opened the pull request:
## Fixes Issue

Closes #2197

## Changes proposed


## Screenshots

## Note to reviewers

nelsonuprety1 commented:
@Anmol-Baranwal @CBID2 @rupali-codes

CBID2 reviewed (dismissed):
LGTM

Anmol-Baranwal reviewed (changes requested):
I don't see a subcategory with text-formatting.

![image](https://github.com/rupali-codes/LinksHub/assets/74038190/10c8bd61-5da8-4848-8bfe-1c44ec59b80f)

You need to add files as per contributing guidelines, just adding a json file won't create a subcategory.

nelsonuprety1 commented:
Hi @Anmol-Baranwal thank you for the reply. So, should i create a new subcategory Text Formatting inside the Other category?

nelsonuprety1 commented:
@Anmol-Baranwal  I have fixed the issues and added the subcategory properly. Please kindly check this PR thank you.
<img width="1440" alt="Screenshot 2023-12-01 at 23 03 51" src="https://github.com/rupali-codes/LinksHub/assets/25173636/c2147a3b-dbbf-4ff2-a609-0eba5aa51afc">

Anmol-Baranwal reviewed (dismissed):
🚀 Looks good to me. 

Thanks for the changes :)

> @rupali-codes @aftabrehan @CBID2 Please review :)

aftabrehan commented on database/other/text-formatting.json:
```diff
@@ -0,0 +1,7 @@
+[{
+    "name": "Text Tune",
+    "description": "Text tune is the ultimate solution for all the text formatting needs!The website provides text formatting features like UPPERCASE, lowercase, Title Case, Reverse Text, Remove WhiteSpace and another feature of this website is to provide with synonyms and antonyms for the word given by the user. If you are looking to format your text or get synonyms and antonyms for given words  ...
```
```suggestion
    "description": "Text tune is the ultimate solution for all the text formatting needs! The website provides text formatting features like UPPERCASE, lowercase, Title Case, Reverse Text, Remove WhiteSpace and another feature of this website is to provide synonyms and antonyms for the word given by the user.",
```

> If you are looking to format your text or get synonyms and antonyms for given words quickly, this website can be helpful.

<details><summary>Why remove this?</summary>
This sentence isn't required, it just reference to the "synonyms and antonyms" feature again.



</p>
</details>

aftabrehan reviewed (changes requested):
@nelsonuprety1, Thanks for the contribution. It could be useful.

Could you please help to adjust the text?

nelsonuprety1 reviewed (commented):
I have adjusted the text as per you request. Thank You.
